index.html

```diff
@@ -0,0 +1,199 @@
+                                </p>
+                            </div>
+                        </li>
+                        <li class="program-list-item">
+                            <div class="program-list-item-container">
+                                <div class="program-list-item-container__icon">
+                                    <i class="fas fa-palette"></i>
+                                </div>
+                                <h2 class="program-list-item-container__title">
+                                    BC Ignites
+                                </h2>
+                                <p class="program-list-item-container__text">
+                                    Get opporunites to network with block chain  around the world, also after the summit.
+                                </p>
+                            </div>
+                        </li>
+                    </ul>
+                    <button class="btn btn-large" type="button">JOIN BLOCK CHAIN SUMMIT 2021</button>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="container">
+              <div class="speakers">
+                  <h1 class="speakers-heading">Guest speakers</h1>
+                    <ul class="speakers-list" id="speakers-list">
+                        
+                    </ul>
+               </div>
+          </div>
+        </section>
+        <section>
+            <div class="container bg-dark">
+                <div class="partner">
+                    <h1 class="partner-heading">Sponsors</h1>
+                    <ul class="partners-list">
+                        <li class="partner-list-item">
+                            <i class="fab fa-bitcoin"></i>
+                        </li>
+                        <li class="partner-list-item">
+                            <i class="fab fa-ethereum"></i>
+                        </li>
+                        <li class="partner-list-item">
+                            <i class="fab fa-amazon"></i>
+                        </li>
+                        <li class="partner-list-item">
+                            <i class="fab fa-airbnb"></i> 
+                        </li>
+                        <li class="partner-list-item">
+                            <i class="fab fa-google"></i> 
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </section>
+        
+    </main>
+    <footer>
+        <div class="container">
+            <div class="footer">
+                <div class="logo">
+                    <a href="#">
+                        <i class="fab fa-bitcoin">Bitcoin</i> 
+                    </a>
+                </div>
+                <p class="rights">
+                    @ 2021 BLOCKCHAIN SUMMIT.
+                </p>
+            </div>
+        </div>
+    </footer>
+    <script src="app.js"></script>
+</body>
+</html>
```

Comment:
- [ ] There's at least 1, but no more than 1, h1 tag in each page.
- [ ] The page is accessible, images contain alt attributes and there's not images or elements blocking the text in both screen sizes.